test/tests/category-plot-tutorial.js

```diff
@@ -21,11 +21,15 @@ var beakerPO;
 
 describe('Category Plots (Bar Charts)', function () {
 
-  beforeEach(function () {
+  beforeEach(function (done) {
     beakerPO = new BeakerPageObject();
-    browser.get(beakerPO.baseURL + "beaker/#/open?uri=file:config%2Ftutorials%2FCategoryPlot.bkr&readOnly=true");
-
-    browser.waitForAngular();
+    browser.get(beakerPO.baseURL + "beaker/#/open?uri=file:config%2Ftutorials%2FCategoryPlot.bkr&readOnly=true").then(
+        function(){done();},
+        function(reason){ console.log('error occurred while browser load page : ' + reason);}
+    );
+    browser.waitForAngular().then(function() {
+      done();
```

Comment:
No need to wrap the done() call in a function. We can just reference 'done' directly:

    browser.waitForAngular().then(done);